src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -110,7 +110,7 @@ RUN for iter in {1..10}; do \
     (exit $exit_code)
 {{/ubuntu}}
 {{#wolfi}}
-RUN apk --no-cache add bash curl fontconfig libstdc++ freetype nss findutils shadow
+RUN apk --no-cache add bash curl fontconfig libstdc++ nss findutils shadow
```

Comment:
freetype was recently removed from the ubuntu base.  fontconfig will pull it in for us.